backend/src/main/java/com/bakdata/conquery/resources/admin/rest/RoleResource.java

```diff
@@ -35,19 +37,17 @@ public Response postRole(Role role) throws JSONException {
 		return Response.ok().build();
 	}
 
-	@POST
-	@Consumes(MediaType.APPLICATION_JSON)
-	public Response postRoles(@NotEmpty List<Role> roles) {
-		processor.addRoles(roles);
-		return Response.ok().build();
-	}
-
 	@GET
-	@Produces(MediaType.APPLICATION_JSON)
 	public Collection<Role> getRoles() {
 		return processor.getAllRoles();
 	}
 
+	@Path("{" + ROLE_ID + "}")
+	@GET
+	public Response getRole(@PathParam(ROLE_ID) Role role) throws JSONException {
```

Comment:
direkt role?

Reply:
Das kommt doch durch das Feature das du implementiert hast, dass Ids gleich zu ihren Objekten aufgelöst werden.

Reply:
Ich meinte in der Response, das wird ja zu Ok gemapped, wenn du keine Exception rausgibst

Reply:
Ah das meinst du. Passe ich noch an